carbon-design-system/carbon-addons-iot-react
# fix(DateTimePickerV2): show invalid message when time fields are empty

Closes #3691 

**Summary**

-when switching from relative to absolute, the time picker fields are empty without any warning message. Apply button is disabled. We should enable warning message to let user know that time fields are empty and they should fill in time value before they can click on apply button. 

**Change List (commits, features, bugs, etc)**

- DateTimePickerV2WithTimeSpinner.jsx
- DateTimePickerV2WithoutTimeSpinner.jsx

**Acceptance Test (how to verify the PR)**

Make sure warning message show up when time fields are empty
- go to [Default](https://deploy-preview-3697--carbon-addons-iot-react.netlify.app/?path=/story/2-watson-iot-experimental-%E2%98%A2%EF%B8%8F-datetimepickerv2--default) story
- click on calendar icon to open drop down preset list
- click on custom range
- select Absolute radio button
- check the time fields are showing warning and apply button is disabled

https://user-images.githubusercontent.com/24279794/213009333-5475336e-3998-48c2-a993-efc1b2a598de.mov



**Regression Test (how to make sure this PR doesn't break old functionality)**

- tests here



**Things to look for during review**
